src/Container.php

```diff
@@ -122,7 +123,9 @@ public function getDeleteShoppingItemCommand(): DeleteShoppingItemCommand
     {
         $repository = $this->getShoppingListRepository();
 
-        return new DeleteShoppingItemCommand($repository);
+        $validator = new DeleteShoppingItemValidator();
```

Comment:
I added the rules and looking at it again since yesterday it make sense that I needed to pass it in. I passed in the rules but PHP Storm didn't have any options to shorten the path name for ShoppingListSlugDoesNotExist and was wondering if I did something wrong there?